README.md

```diff
@@ -206,6 +206,11 @@ To try out SynapseML using the R autogenerated wrappers [see our
 instructions](website/docs/reference/R-setup.md).  Note: This feature is still under development
 and some necessary custom wrappers may be missing.
 
+### C# (.NET)
+
+To try out SynapseML with .NET, please follow [instructions](website/docs/reference/dotnet-setup.md).
+Note: This feature is not fully supported so some special functions may be missing.
```

Comment:
We might want to use less scary language here. Do we have a list of unsupported stages we can include in the target document?